commit 0478c46c6d0b1b58fd77dc5363fcf3533552f41b (refs/changes/79/2879/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-10-27T15:05:19-07:00 (1 year, 11 months ago)
    
    OS-6425 loader should ignore active multi_vdev_crash_dump feature on zpool

diff --git a/usr/src/boot/sys/boot/zfs/zfsimpl.c b/usr/src/boot/sys/boot/zfs/zfsimpl.c
index f34f88d88e..26654c0469 100644
--- a/usr/src/boot/sys/boot/zfs/zfsimpl.c
+++ b/usr/src/boot/sys/boot/zfs/zfsimpl.c
@@ -59,6 +59,7 @@ static const char *features_for_read[] = {
 	"org.open-zfs:large_blocks",
 	"org.illumos:sha512",
 	"org.zfsonlinux:large_dnode",
+	"com.joyent:multi_vdev_crash_dump",
 	NULL
 };
 
